src/community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/configuration/SupportedFormat.java

```diff
@@ -23,11 +24,10 @@ public String getFormat() {
     }
 
     public static List<SupportedFormat> getByExtension(String extensions) {
-        if (extensions == null) {
-            return Arrays.asList(SupportedFormat.values());
-        } else if (extensions.equals("xml")) {
-            return Arrays.asList(GML);
-        } else return Arrays.asList(JSONLD, GEOJSON);
+        if (extensions == null) return Arrays.asList(SupportedFormat.values());
+        else if (extensions.equals("xml")) return Arrays.asList(GML);
+        else if (extensions.equals("xhtml")) return Arrays.asList(HTML);
+        else return Arrays.asList(JSONLD, GEOJSON);
```

Comment:
This code formatting looks really weird.

Reply:
This is how the maven formatting plugin formats it.